src/geometryVparMu/geometry/geometry.hpp

```diff
@@ -195,7 +195,7 @@ template <class ElementType>
 using ConstFieldSpVpar = ConstField<ElementType, IdxRangeSpVpar>;
 using DConstFieldSpVpar = ConstFieldSpVpar<double>;
 
-/* A OTER
+/* A ôter
```

Comment:
```suggestion
/* To remove
```
But maybe this can be removed now @gdgirard ?

Reply:
removed in c657ad4b8144a1798193a327fe30ba9f612eb97d